src/index.ts

```diff
@@ -59,6 +64,10 @@ export interface ServerlessClamscanProps {
    * An optional list of S3 buckets to configure for ClamAV Virus Scanning; buckets can be added later by calling addSourceBucket.
    */
   readonly buckets?: IBucket[];
+  /**
+   * An optional list of additional S3 buckets properties for creating event notifications
+   */
+  readonly bucketsEventNotificationProps?: BucketEventNotificationProps[];
```

Comment:
**suggestion:** a common convention in CDK is to call something like this _options_, as they are optional and additive to already existing `buckets` prop.

```suggestion
  readonly bucketsEventNotificationOptions?: BucketEventNotificationProps[];
```

I think it reads better when you actually use it:

```ts
new ServerlessClamscan(this, `${props.prefix}serverless-clamscan`, {
  buckets: [bucket],
  bucketsEventNotificationOptions: [
    {
      bucketName: bucket.bucketName,
      notificationKeyFilters: [{ suffix: ".gif" }],
    }
  ],
});
```